nodeapi-spec.yaml

```
@@ -714,7 +714,106 @@ paths:
                       $ref: "#/components/schemas/ExperimentNote"
                   offset:
                     type: integer
+    post:
+      summary: Experiment activity note CREATE
```

The capitalisation here is different, can we change it to Experiment Comments instead?
```suggestion
      summary: Experiment Comment CREATE
```